.env

```diff
@@ -0,0 +1,4 @@
+SKIP_PREFLIGHT_CHECK=true
+PORT=8081
+BROWSER=false
+REACT_APP_DHIS2_BASE_URL=http://dev2.eyeseetea.com:8082
```

Comment:
Not sure about this... either we remove it or we point to 8085

Reply:
Let's remove it.